controllers/scheme.go

```diff
@@ -0,0 +1,18 @@
+package controllers
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+
+	etcdv1alpha1 "github.com/improbable-eng/etcd-cluster-operator/api/v1alpha1"
+)
+
+var (
+	scheme = runtime.NewScheme()
+)
+
+func init() {
+	_ = clientgoscheme.AddToScheme(scheme)
+	_ = etcdv1alpha1.AddToScheme(scheme)
+
+}
```

Comment:
This is moved here because tests were modifying the k8s.io/client-go scheme in place, which wasn't right.

Reply:
I don't think I quite understand what you mean. Modifying it in place?

Reply:
It was modifying the global scheme declared by client-go, rather than adding that scheme to a local scheme.
That problem was fixed in https://github.com/kubernetes-sigs/kubebuilder/pull/809 but it looks like they forgot to make the same change to the skaffolded integration tests.

Reply:
I'll revert this anyway, because I didn't need the scheme in `actions.go` in the end.